Formula/i/infra-arcana.rb

```diff
@@ -0,0 +1,70 @@
+class InfraArcana < Formula
+  desc "Roguelike set in the early 20th century"
+  homepage "https://sites.google.com/site/infraarcana/"
+  url "https://gitlab.com/martin-tornqvist/ia/-/archive/v22.1.0/ia-v22.1.0.tar.gz"
+  sha256 "3a2d05ece62fdef51e8515004f0412ac825b0621ab81dc918240c6c04d68c96f"
+  license "AGPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "sdl2"
+  depends_on "sdl2_image"
+  depends_on "sdl2_mixer"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+
```

Comment:
Does this work?
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
It looks like this works, adjusted accordingly in https://github.com/Homebrew/homebrew-core/pull/151809/commits/772913fcb75e160c4cb08c0dedfad7c7acbcb908, thanks!